pkg/sloop/webserver/webserver.go

```diff
@@ -90,25 +91,28 @@ func logWebError(err error, note string, r *http.Request, w http.ResponseWriter)
 
 // Example input: r.URL=/webfiles/static/style.css
 // Returns file: <webFiles>/static/style.css
-func webFileHandler(w http.ResponseWriter, r *http.Request) {
-	fixedUrl := strings.TrimPrefix(fmt.Sprint(r.URL), "/webfiles")
-	if strings.Contains(fixedUrl, "..") {
-		logWebError(nil, "Not allowed", r, w)
-		return
-	}
-	fullPath := path.Join(prefix, fixedUrl)
-	data, err := readWebfile(fullPath, &afero.Afero{afero.NewOsFs()})
-	if err != nil {
-		logWebError(err, "Error reading web file: "+fixedUrl, r, w)
-		return
-	}
-	w.Header().Set("content-type", mime.TypeByExtension(filepath.Ext(fullPath)))
-	_, err = w.Write(data)
-	if err != nil {
-		logWebError(err, "Error writing web file: "+fixedUrl, r, w)
-		return
+func webFileHandler(currentContext string) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		webFilesPathToTrim := "/" + currentContext + "/webfiles"
```

Comment:
what if currentContext is "" ?

Reply:
Its the context for which Sloop is deployed. It would always be there.

Reply:
added unit test